packages/server/modules/gatekeeperCore/domain/billing.ts

```diff
@@ -75,7 +23,7 @@ export type PaidWorkspacePlan = BaseWorkspacePlan & {
 }
 
 export type TrialWorkspacePlan = BaseWorkspacePlan & {
-  name: TrialWorkspacePlans
+  name: TrialEnabledPaidWorkspacePlans
```

Comment:
These also constrain the statuses each workspace plan can be on, maybe that is also usefull in the frontend.
cc @Mikehrn

Reply:
The FE is likely going to work with the WorkspacePlan GQL type directly. Casting it to these would require helpers/type guards (cause the types don't fit 1:1) and the FE use cases for these aren't immediately obvious to me. We can always move them to shared later, but if Mike sees a use for these, I can move em

Reply:
This will be any obsolete in a few weeks right, as we will abandon the concept of trials existing?